dash/development/component_loader.py

```diff
@@ -94,8 +97,15 @@ def generate_classes(namespace, metadata_path='lib/metadata.json'):
             componentData['description'],
             namespace
         )
+        generate_class_file_r(
+            name,
+            componentData['props'],
+            componentData['description'],
+            namespace
+        )
 
         # Add an import statement for this component
+        # RK: need to add import statements in R namespace file also
```

Comment:
Is that still todo ?

Reply:
I've added `export()` statements to the `NAMESPACE` file for the R package, don't think we'll need to provide an import statement. Will remove the comment.